src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Ball extends ColorSupplier{
+    public Colors color = getRandomColor();
+    public int number = new Random().nextInt(100);
+
```

Comment:
```suggestion
    public Colors color;
    public int number;
```
this values should be initialized from Lottery